packages/lib/src/vis/rgb/RgbVis.tsx

```diff
@@ -59,12 +58,18 @@ function RgbVis(props: Props) {
           isIndexAxis: true,
           flip: true,
         }}
-        interactionKeys={DEFAULT_INTERACTIONS_KEYS}
+        interactionKeys={{
+          Pan: true,
+          Zoom: true,
+          XAxisZoom: { modifierKey: 'Alt', disabled: keepRatio },
+          YAxisZoom: { modifierKey: 'Shift', disabled: keepRatio },
+          SelectToZoom: { modifierKey: 'Control' },
+        }}
```

Comment:
I could not use `DEFAULT_INTERACTIONS_KEYS` here due to dynamic setting of `disabled`.

Do you think it is worth putting the creating of an object in an utility ?